### PR TITLE
FLZ-7915_Error-when-submitting-Form-via-Embedded-board

### PR DESCRIPTION
**Pull request by maor-folloze**

### CODE REVIEW

File: src/liveboard/Liveboard.ts [hunk 1]
```diff
@@ -698,6 +698,7 @@ export class Liveboard {
         return this.fetchService.withDisableOnPreview((): Promise<CtaResponseV1> => {
             return new Promise((resolve, reject) => {
                 return this.notifyIdentity(boardId, options).then(_ => {
+                    console.log("blabla", _);
```
Comment (maor-folloze) [inline src/liveboard/Liveboard.ts:701]: ops
will be removed

**Comment by TanyaFolloze** [comment]
Task linked: [FLZ-7915 Error when submitting Form via Embedded board](https://app.clickup.com/t/3749831/FLZ-7915)

**Review by adi-heffer-folloze — APPROVED** [review]

File: src/common/FetchService.ts [hunk 2]
```diff
@@ -174,13 +174,22 @@ export class FetchService {
         return Promise.reject(error);
     }
 
+    private withCredentials = (request: AxiosRequestConfig): AxiosRequestConfig => {
```
Comment (rotemgrim) [inline src/common/FetchService.ts:177]: why not putting this under the request interceptor on line 194?

File: src/common/FetchService.ts [hunk 3]
```diff
@@ -174,13 +174,22 @@ export class FetchService {
...
+        if (this.isEmbeddedRequest && !request.url?.includes(this.options.analyticsServiceEndpoint)) {
```
Comment (rotemgrim) [inline src/common/FetchService.ts:178]: why not sending to analytics service also?

**Review by rotemgrim — CHANGES_REQUESTED** [review]

**Review by rotemgrim — APPROVED** [review]

**Pull request merged**